qpmodel/Catalog.cs

```diff
@@ -55,7 +55,8 @@ public class ColumnDef
         public override string ToString() => $"{name_} {type_} [{ordinal_}]";
     }
 
-    public class Distribution {
```

Comment:
How about split the patch into two PRs? One is for migration itself and the other is for code cleanup. In this way, it is easier to follow the changes.

Reply:
Actually I've already split the changes into two commits. You can just look at the first commit to see the actual changes.

I'm also happy to split this one into two PRs if you think that is better.

Reply:
Sure - I am good with two commits as well.